fiftyone/utils/cvat.py

```diff
@@ -3431,19 +3443,27 @@ def __init__(self, name, url, username=None, password=None, headers=None):
         self._password = password
         self._headers = headers
 
+        self._server_version = None
```

Comment:
FYI- `pylint` should have given you a warning here that all variables should be defined in the constructor for readability